_posts/2017-04-21-calculating-the-importance-of-an-artwork-with-apache-spark.markdown

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Calculating the Importance of an Artwork with Apache Spark"
+date: 2017-04-21
+comments: true
+author: db
+categories: [Big Data, Apache Spark]
+---
+How important is a single artwork within the artist's body of work? At Artsy we try to answer this question by extracting and scoring a set of features across roughly one million artworks. We call it an _iconicity_ score and we calculate that in Apache Spark in just under 5 minutes.
```

Comment:
I feel it's slightly ambiguous here what the 5 minutes refers to — at first glance I read it as 5 minutes to calculate the iconicity of a single artwork, but you mean it took 5 minutes to calculate the iconicity for _all_ ~1M artworks, yes?